services/core/shared/core/compat/sdk_v5/network.js

```diff
@@ -0,0 +1,26 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2019 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+const coreApi = require('./coreApi');
+
+const getNetworkStatus = async () => {
+	const core = await coreApi();
+	const status = await core.getNetworkStatus();
```

Comment:
This line of code isn't necessary with the currently recommended change in `services/core/shared/core/compat/sdk_v5/coreApi.js`.
```suggestion
const coreApi = require('./coreApi');

const getNetworkStatus = async () => {
	const status = await coreApi.getNetworkStatus();
```

Reply:
Done!